src/server/pkg/pfsdb/pfsdb.go

```diff
@@ -46,6 +47,17 @@ func RepoRefCounts(etcdClient *etcd.Client, etcdPrefix string) col.Collection {
 	)
 }
 
+// PutFileRecords returns a collection of putFileRecords
+func PutFileRecords(etcdClient *etcd.Client, etcdPrefix string) col.Collection {
+	return col.NewCollection(
+		etcdClient,
+		path.Join(etcdPrefix, putFileRecordsPrefix),
+		nil,
+		nil,
+		nil,
+	)
```

Comment:
I'd recommend adding a template to this even though you don't use indexes. In the commit invariants branch that will be merged in soon we use the template to make sure that you don't accidentally put one type in the collection and read out a different one. It's just an added safety check that I think is nice to have.